src/lib/AnnotationControls.ts

```diff
@@ -84,21 +93,48 @@ export default class AnnotationControls {
     private handleFullscreenExit = (): void => this.handleFullscreenChange(false);
 
     /**
-     * Region comment button click handler
+     * Deactive current control button
      */
-    private handleRegionClick = (onRegionClick: RegionHandler) => (event: MouseEvent): void => {
+    public deactivateCurrentControl = (): void => {
+        if (!this.currentActiveControl) {
+            return;
+        }
+
+        const updateButton = this.controlsMap[this.currentActiveControl];
+
+        this.currentActiveControl = null;
+        updateButton();
+    };
+
+    /**
+     * Update region button UI
+     */
+    private updateRegionButton = (): void => {
         const regionButtonElement = this.controlsElement.querySelector(`.${CLASS_REGION_BUTTON}`);
 
         if (regionButtonElement) {
-            this.isRegionActive = !this.isRegionActive;
-            if (this.isRegionActive) {
+            if (this.currentActiveControl === ANNOTATION_MODE.region) {
                 regionButtonElement.classList.add(CLASS_BUTTON_ACTIVE);
             } else {
                 regionButtonElement.classList.remove(CLASS_BUTTON_ACTIVE);
             }
         }
+    };
+
+    /**
+     * Region comment button click handler
+     */
+    private handleRegionClick = (onRegionClick: RegionHandler) => (event: MouseEvent): void => {
+        const activeControl = this.currentActiveControl;
```

Comment:
The call to this.deactivateCurrentControl() seems unnecessary. It seems like this method's responsibility should be (and all `handle([Region|Highlight|Draw])Click` for that matter):

- update `currentActiveControl`
- update all buttons states accordingly
- callback to Viewer with the type toggled

so maybe something like this
```
this.currentActiveControl = this.currentActiveControl === 'region' ? 'none' : 'region';
this.controlsMap.keys().forEach(updateButton => updateButton());
onModeClick('region');
```
Thoughts?